dts/arm64/intel_socfpga/intel_socfpga_agilex.dtsi

```diff
@@ -50,6 +50,12 @@
 		label = "sysmgr";
 	};
 
+	rstmgr: rstmgr@ffd11000 {
+		compatible = "syscon";
+		reg = <0xffd11000 0x1000>;
+		label = "rstmgr";
+	};
```

Comment:
Any reason to not have `sysmgr` and `rstmgr` under a single `syscon` node? And are you sure that using a `syscon` here is the correct choice?

The `syscon` region is really a shortcut for when you have a region containing a set of miscellaneous registers (for example clock + reset + misc, etc.. registers in the same region). When in the region you have several registers that are not miscellaneous but instead they have a very precise usage, in that case using `syscon` is not the correct choice.

Now, my question here is: are you using a `syscon` for the reset manager because Zephyr is not shipping any reset controller?